### PR TITLE
[LP-11296] Remove deprecated code

### PR DESCRIPTION
**Pull request by milos1290**
What              | Where/Who
------------------|----------------------------------------
JIRA Issue        | [LP-11296](https://leanplum.atlassian.net/browse/LP-11296)
People Involved   | @milos1290

### CODE REVIEW

File: Example/Tests/Classes/LPInboxTest.m [hunk 1]
```diff
@@ -427,18 +316,23 @@ - (void)test_onForceContentUpdate {
...
+                                         userInfo:@{@"FCU": @"Inbox"}];
         return [HTTPStubsResponse responseWithError:error];
     }];
+
+    NSLog(@"XXX: Before");
     
     XCTestExpectation *responseExpectation3 = [self expectationWithDescription:@"response3"];
-    [[Leanplum inbox] onForceContentUpdate:^void(BOOL success){
+    [[Leanplum inbox] onForceContentUpdate:^(BOOL success) {
+        NSLog(@"XXX: FCU FAIL");
```
Comment (e7mac) [inline Example/Tests/Classes/LPInboxTest.m:328]: different log?

File: Leanplum-SDK/Classes/Internal/Leanplum.m [hunk 2]
```diff
@@ -37,20 +37,19 @@
...
 #import "LPUtils.h"
 #import "LPAppIconManager.h"
-#import "LPUIEditorWrapper.h"
 #import "LPCountAggregator.h"
 #import "LPRequestFactory.h"
 #import "LPFileTransferManager.h"
 #import "LPRequestSender.h"
 #import "LPAPIConfig.h"
 #import "LPOperationQueue.h"
+#include <sys/sysctl.h>
```
Comment (e7mac) [inline Leanplum-SDK/Classes/Internal/Leanplum.m:52]: is this needed?

File: Example/Tests/Classes/LPInboxTest.m (hunk 1 above)
Comment (milos1290) [reply Example/Tests/Classes/LPInboxTest.m:328]: will be removed when tests are fixed

File: Leanplum-SDK/Classes/Internal/Leanplum.m (hunk 2 above)
Comment (milos1290) [reply Leanplum-SDK/Classes/Internal/Leanplum.m:52]: yeah

**Pull request merged**